libs/components/modals/src/lib/modules/confirm/confirm.service.ts

```diff
@@ -21,16 +22,22 @@ import { SkyConfirmComponent } from './confirm.component';
 })
 export class SkyConfirmService {
   #modalService: SkyModalService;
+  #logService: SkyLogService;
 
-  constructor(modalService: SkyModalService) {
+  constructor(modalService: SkyModalService, logService: SkyLogService) {
```

Comment:
Don't hate me - can we move this to the `SkyConfirmComponent` constructor? These services are often mocked by our consumers and adding a constructor parameter to them can cause unit test issues for consumers (we have unfortunately ran into this a lot). So we either need to move this to the component which avoids the issue all together or make this `@Optional` so that consumers can skip it (and then handle that properly where we call the `deprecate` function).

Reply:
removed for now, changes pushed to branch `log-deprecated-properties`